tests/common.go

```diff
@@ -3157,3 +3181,173 @@ func init() {
 	logrus.StandardLogger().Hooks.Add(log.NewHook())
 	logrus.SetOutput(os.Stdout)
 }
+
+//CreateJiraIssueWithLogs creates a jira issue and copy logs to nfs mount
+func CreateJiraIssueWithLogs(issueDescription, issueSummary string) {
+	issueKey := jirautils.CreateIssue(issueDescription, issueSummary)
+	if issueKey != "" {
+		collectAndCopyDiagsOnWorkerNodes(issueKey)
+		collectAndCopyStorkLogs(issueKey)
+		collectAndCopyOperatorLogs(issueKey)
+		collectAndCopyAutopilotLogs(issueKey)
+
+	}
+
+}
+
+func collectAndCopyDiagsOnWorkerNodes(issueKey string) {
+	isIssueDirCreated := false
+	for _, currNode := range node.GetWorkerNodes() {
+		err := runCmd("pwd", currNode)
+		if err == nil {
+			logrus.Infof("Creating directors logs in the node %v", currNode.Name)
+			runCmd("mkdir /root/logs", currNode)
+			logrus.Info("Mounting nfs diags directory")
+			runCmd("mount -t nfs diags.pwx.dev.purestorage.com:/var/lib/osd/pxns/688230076034934618 /root/logs", currNode)
```

Comment:
Store these in constants at the beginning of the program or get them from the config map, otherwise you will have to look in the code if this changes in the future.

Reply:
done